script/Arcueid/Day 14/14_00_ARC14_1.txt

```diff
@@ -1152,7 +1154,7 @@ I wanted to feel more of her warmth.
 [sha:e0f85177c7e4c4b4c4db5470cd030b52843a1849]{
 -- Page 69, Offset 36970.
 -- もっと――――もっと、あの笑顔を、見ていたかった。
-I wanted――――to see more of that smile.
+I wanted――to see more of that smile.
```

Comment:
This dash has no purpose as is. We can create some purpose that likens the JP, I guess?
>I wanted to see more――so much more――of that smile.